ledger/light/integration/operations_test.go

```diff
@@ -74,7 +74,11 @@ func CallGetCode(ctx context.Context, s *Server, id insolar.ID) payload.Payload
 	return nil
 }
 
-func MakeSetIncomingRequest(objectID, reasonID insolar.ID, isCreation, isAPI bool) (payload.SetIncomingRequest, record.Virtual) {
+func MakeSetIncomingRequestFromAPI(objectID, reasonID insolar.ID, isCreation bool) (payload.SetIncomingRequest, record.Virtual) {
```

Comment:
This is not necessary, we should consider simplification at a later time thought.

Reply:
done